apis/Google.Cloud.Firestore/Google.Cloud.Firestore.Tests/QueryTest.cs

```diff
@@ -1000,7 +1000,7 @@ public void Equality_Where()
                     col.WhereEqualTo("otherfield", 10), // Different field
                     col.WhereLessThan("field",  10), // Different operator
                     col.WhereEqualTo("field", 20), // Different value
-                    col.Select("a") // query without any filter(query._filters = null).
+                    col.Select("a") // query without any filter (query._filters is null).
```

Comment:
Sorry to keep nitting on this, but you've still got a lower-case q at the start...

Reply:
Oh missed that will update, I am removing the `do not merge` label than